File: baekjoon/13000/13420.py
T = int(input())
for t in range(T):
    s = input().split(" ")
    b = int(s[-1])
    a1, a2 = int(s[0]), int(s[2])
    if s[1] == '+':
        a = a1 + a2
    elif s[1] == '-':
        a = a1 - a2
    elif s[1] == '*':
        a = a1 * a2
    else:
        a = a1 // a2
    if a == b:
        print("correct")
    else:
        print("wrong answer")

File: baekjoon/13000/13420.rs
use std::io::*;

struct Scanner {
    it: std::str::SplitAsciiWhitespace<'static>,
}

impl Scanner {
    fn new() -> Self {
        let mut s = String::new();
        stdin().read_to_string(&mut s).ok();
        Self { it: s.leak().split_ascii_whitespace() }
    }
    fn read<T: std::str::FromStr>(&mut self) -> T {
        self.it.next().unwrap().parse::<T>().ok().unwrap()
    }
}

fn main() {
    let mut sc = Scanner::new();
    let mut bw = BufWriter::new(stdout().lock());
    let t = sc.read::<usize>();
    for _ in 0..t {
        let a = sc.read::<i64>();
        let op = sc.read::<char>();
        let b = sc.read::<i64>();
        let _ = sc.read::<char>();
        let c = sc.read::<i64>();
        let d = match op {
            '+' => a + b,
            '-' => a - b,
            '*' => a * b,
            '/' => a / b,
            _ => unreachable!(),
        };
        writeln!(bw, "{}", if c == d { "correct" } else { "wrong answer" }).ok();
    }
}
